.github/workflows/sonarcloud.yml

```
@@ -20,26 +20,28 @@
-      #     pip install pytest coverage
-      #     pytest --cov=backend --cov-report xml  # Backend coverage report for Python
-
+      # Install Python dependencies and run backend tests
+      - name: Install Python dependencies and run backend tests
+        run: |
+          cd backend
+          pip install -r requirements.txt
+          pip install pytest coverage
+          pytest tests --cov=. --cov-report=xml --cov-report=term-missing  # Backend coverage report for Python
+        env:
+           DATABASE_URL: "sqlite:///:memory:"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 11

(indentation)